### PR TITLE
Bishal

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by shivraj110 — APPROVED** [review]

File: BasicGit/a.html [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Learning GitHub</title>
 </head>
 <body>
+        <h1> ASP.NET CORE Course Started From Today.</h1>
+
```
Comment (chandan-g-bhagat) [inline BasicGit/a.html:8]: remove this blank line

**Review by thechandanbhagat — APPROVED** [review]

**Review by Bikki-99 — APPROVED** [review]

**Review by thechandanbhagat — APPROVED** [review]

**Review by sazal11 — APPROVED** [review]

**Review by dineslama — APPROVED** [review]

**Review by imanojkarki — APPROVED** [review]
well done

**Review by imanojkarki — APPROVED** [review]
wow...excellent work - keep it up

**Review by imanojkarki — APPROVED** [review]
super duper - welldone

**Pull request merged**